middleware/handleRedirects.ts

```diff
@@ -16,32 +15,38 @@ export default async function ({ $config, route, redirect }) {
     path = path.slice(6)
   }
 
-  // if this is a BCSC signin, redirect to BCSC URL
+  // if this is a BCSC signin, navigate to BCSC URL
   if (path.startsWith('/signin/bcsc')) {
     // redirect to Auth signin page + set URL to go to after login
     const loginUrl = getLoginUrl() || $config.registryDashboard
     const url = $config.signinBcscUrl + encodeURIComponent(loginUrl)
-    return redirect(url)
+    window.location.assign(url)
```

Comment:
I tested `redirect()` in other places in this project and all of them routed correctly, even to external URLs. This file is special for some reason.